sample-app/src/components/FilterSearch.tsx

```diff
@@ -0,0 +1,125 @@
+import { SearchParameterField } from "@yext/answers-headless";
+import { useState } from "react";
+import { useAnswersActions } from '@yext/answers-headless-react';
+import InputDropdown from "./InputDropdown";
+import renderWithHighlighting from "./utils/renderWithHighlighting";
+import { Option } from "./DropdownSection";
+import DropdownSection from "./DropdownSection";
+import { processTranslation } from "./utils/processTranslation";
+import { useSynchronizedSearch } from "../hooks/useSynchronizedSearch";
+
+const SCREENREADER_INSTRUCTIONS = 'When autocomplete results are available, use up and down arrows to review and enter to select.'
+
+export interface FilterSearchProps {
+  title: string,
+  sectioned: boolean,
+  searchFields: Omit<SearchParameterField, 'fetchEntities'>[],
+  screenReaderInstructionsId: string
+}
+
+export default function FilterSearch (props: FilterSearchProps): JSX.Element {
+  const { title, sectioned, searchFields, screenReaderInstructionsId } = props;
+  const answersActions = useAnswersActions();
+  const [input, setInput] = useState('');
+  const searchParamFields = searchFields.map((searchField) => {
+    return { ...searchField, fetchEntities: false }
+  });
+
+  const [filterSearchResponse, executeFilterSearch] = useSynchronizedSearch((inputValue) => {
+    return inputValue
+      ? answersActions.executeFilterSearch(inputValue, sectioned, searchParamFields)
+      : answersActions.executeFilterSearch('', sectioned, searchParamFields);
+  });
+
+  let sections: { results: Option[], label?: string }[] = [];
+  if (filterSearchResponse?.results) {
+    sections = filterSearchResponse.sections.map(section => {
+      return {
+        results: section.results.map(result => {
+          return {
+            value: result.value,
+            display: renderWithHighlighting(result)
+          };
+        }),
+        label: section.label
+      };
+    });
+  }
+
+  sections = sections.filter(section => section.results.length > 0);
+
+  let screenReaderText = processTranslation({
+    phrase: `0 autocomplete option found.`,
+    pluralForm: `0 autocomplete options found.`,
+    count: 0
+  });
+  if (sections.length > 0) {
+    const screenReaderPhrases = sections.map(section => {
+      const optionInfo = section.label 
+        ? `${section.results.length} ${section.label}`
+        : `${section.results.length}`;
+      return processTranslation({
+        phrase: `${optionInfo} autocomplete option found.`,
+        pluralForm: `${optionInfo} autocomplete options found.`,
+        count: section.results.length
+      });
+    });
+    screenReaderText = screenReaderPhrases.join(' ');
+  }
+
+  return (
+    <div className='FilterSearch'>
```

Comment:
I think we want cssClasses specified in the props now, instead of hardcoding them in, like what you have for InputDowndown!

Reply:
Oh ok, so all the cssClasses that we pass in for InputDropdown and DropdownSection should be specified in props for FilterSearch? And similarly for SearchBar?

Reply:
double check this with connor, but yea I believe the css classes should be specified in the parent component

Reply:
Yes, we are going to have all the css classes available in the props. No need to worry about it for the search bar since I will update that in my PR

Reply:
Oops, didn't see the edited comment. I went ahead and changed it in FilterSearch

Reply:
No worries, it will just make our lives easier when we add the tailwind styling